src/Console/ModelsCommand.php

```diff
@@ -82,8 +82,8 @@ public function handle()
         $filename = $this->option('filename');
         $this->write = $this->option('write');
         $this->dirs = array_merge(
-            $this->laravel['config']->get('ide-helper.model_locations'),
-            $this->option('dir')
+            (array) $this->laravel['config']->get('ide-helper.model_locations'),
+            (array) $this->option('dir')
```

Comment:
Why is this necessary?

It's defined as array so it can't be not an array, see https://github.com/barryvdh/laravel-ide-helper/blob/c0bb9ee1fd623f90c2ba593ab151923eb66c933a/src/Console/ModelsCommand.php#L152

Reply:
Probably the array casting is added to ensure it becomes an array. The signature `InteractsWithIO::option(string $key = null): string|array|bool|null` shows it potentially can return many types. I'm not sure if this is a good thing to add though it would break when changing the definition.